wolfcrypt/src/ecc.c

```diff
@@ -5369,7 +5429,34 @@ int wc_ecc_verify_hash_ex(mp_int *r, mp_int *s, const byte* hash,
        return err;
     }
     (void)hashlen;
+#elif defined(WOLFSSL_CRYPTOCELL)
+
+   /* Extract R and S */
+
+   err = mp_to_unsigned_bin(r, &sigRS[0]);
+   if (err != MP_OKAY) {
+       return err;
+   }
+   err = mp_to_unsigned_bin(s, &sigRS[32]);
+   if (err != MP_OKAY) {
+       return err;
+   }
+
+   /* verify the signature using the public key */
+   err = CRYS_ECDSA_Verify(&sigCtxTemp,
+                           &key->ctx.pubKey,
+                           CRYS_ECPKI_AFTER_HASH_SHA256_mode,
+                           &sigRS[0],
+                           sizeof(sigRS),
+                           (byte*)hash,
+                           hashlen);
 
+   if (err != SA_SILIB_RET_OK) {
+       WOLFSSL_MSG("CRYS_ECDSA_Verify failed");
+       return err;
+   }
+   /* valid signature */
+   *res = 1;
```

Comment:
Double check this is always true. If the check happens in `CRYS_ECDSA_Verify` please indicate that.